### PR TITLE
Update normalize_search_engine UDF

### PR DESCRIPTION
**Pull request by benmiroglio**
Updating this UDF per feedback from mconnor and the finance team. Ideally we would backfill this for a few months.

### CODE REVIEW

**Comment by benmiroglio** [comment]
@Ben-Wu  can you take a look at this? I believe you wrote the first one

File: udf/normalize_search_engine.sql [hunk 1]
```diff
@@ -31,5 +26,7 @@ SELECT
   assert_equals('Google', udf.normalize_search_engine('google')),
   assert_equals('Google', udf.normalize_search_engine('Google-abc')),
   assert_equals('Other', udf.normalize_search_engine('not-bing')),
+  assert_equals('Other', udf.normalize_search_engine('other-Google')),
   assert_equals('Other', udf.normalize_search_engine('engine')),
   assert_null(udf.normalize_search_engine(NULL))
+t
```
Comment (Ben-Wu) [inline udf/normalize_search_engine.sql:32]: Should have a new line at the end

File: udf/normalize_search_engine.sql [hunk 2]
```diff
@@ -8,19 +8,14 @@ CREATE OR REPLACE FUNCTION
...
-      WHEN STARTS_WITH(engine, 'yandex')
-      OR STARTS_WITH(engine, 'Yandex')
-      OR STARTS_WITH(engine, 'other-Yandex') THEN 'Yandex'
+      WHEN STARTS_WITH(LOWER(engine), 'google') THEN 'Google'
+      WHEN STARTS_WITH(LOWER(engine), 'ddg') 
+      OR STARTS_WITH(LOWER(engine), 'duckduckgo') THEN 'DuckDuckGo'
+      WHEN STARTS_WITH(LOWER(engine), 'bing') THEN 'Bing'
+      WHEN STARTS_WITH(LOWER(engine), 'yandex')
+      OR STARTS_WITH(LOWER(engine), 'yasearch') THEN 'Yandex'
+      WHEN STARTS_WITH(LOWER(engine, 'amazon')) THEN 'Amazon'
```
Comment (Ben-Wu) [inline udf/normalize_search_engine.sql:17]: `)` is in the wrong place here and the line below

**Comment by benmiroglio** [comment]
filed this bug for the backfill: https://bugzilla.mozilla.org/show_bug.cgi?id=1644520

**Pull request merged**